Makefile

```diff
@@ -171,6 +172,10 @@ check-fast: lint $(PY35) $(PYPY) $(TOX)
 	$(TOX) -e py26-brief
 	$(TOX) -e py35-prettyquick
 
+check-rst: $(RSTLINT)
+	$(RSTLINT) CONTRIBUTING.rst
+	$(RSTLINT) README.rst
```

Comment:
Why lint just these two files? Why not lint the entire `docs` directory while we're at it?